lib/turn.rb

```diff
@@ -0,0 +1,29 @@
+
+class Turn
+  attr_reader   :guess
```

Comment:
I like the formatting there may be some extra spaces but that is all up to you. 

Some people list them all in one line and some people list them the way you did. No right answer here. Good job.